NAMESPACE

```diff
@@ -8,6 +8,10 @@ S3method(all.equal,shade)
 S3method(c,shade)
 S3method(coords,default)
 S3method(coords,shade)
+S3method(lightness,"function")
+S3method(lightness,Scale)
+S3method(lightness,default)
+S3method(lightness,ggproto_method)
```

Comment:
This is going to get pretty complicated if four new methods are needed for each property. If this level of detail is necessary then I think it would have to be baked in a bit deeper.

Reply:
Okay. I'll push an alternative request which goes in at `.replaceProperty` level, so you can compare.